packages/app-bridge/src/types/Emitter.ts

```diff
@@ -113,6 +113,18 @@ export type Emitter = MittEmitter<{
               action: 'delete';
           };
 
+    'AppBridge:GuidelineDocumentPageTargetsAction': {
+        targets: number[];
```

Comment:
the standard pattern is to use action & payload to stay consistent, so the signature, in the end, should be `action: '...'`, and the object that has `{targets: number[], pageIds: number[]} `

Reply:
fixed